src/conversion/from_geo_types.rs

```diff
@@ -1,3 +1,7 @@
+// only enables the `doc_cfg` feature when
+// the `docsrs` configuration attribute is defined
+#[cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
No biggie, so take it or leave it, but can we do this in a single place, like at src/lib.rs or does it need to be duplicated around?

Reply:
I have no idea, but that would be better. I'll ask elsewhere…

Reply:
Tokio does it in the crate root - so seems like it should work.

https://github.com/tokio-rs/tokio/blob/master/tokio/src/lib.rs#L18